packages/docs/locales/en/en.json

```diff
@@ -639,20 +649,34 @@
         "verticalOffset": "Sets the vertical offset of the component from the border of the monitor"
       },
       "events": {
-        "click" :"Emitted when user clicked on backtop"
+        "click": "Emitted when user clicked on backtop"
       },
       "methods": {},
       "slots": {
         "default": "Slot for backtop content"
       }
+    },
+    "VaAppBar": {
```

Comment:
Do you have a follow up on this?

Reply:
Dima is working on it.

Reply:
@SergeiPleshkov so are you both working on same PR currently? o.o

Reply:
@asvae No, it's in a separate PR